graph_creator/static/graph_creator/map.js

```diff
@@ -16,7 +16,7 @@ L.tileLayer('https://api.mapbox.com/styles/v1/mapbox/streets-v10/tiles/256/{z}/{
 edxData.forEach(function(data){
     L.marker([data.fields.latitude, data.fields.longitude])
```

Comment:
It is actually copy-paste from documentation, so we can leave it as it is, but better rename.